features/src/main/feature/camel-features.xml

```diff
@@ -581,22 +581,16 @@
         <feature>jackson</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-chatscript/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-coap' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-buffer/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-handler/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/californium-core/${californium-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector/${californium-version}$overwrite=merge&amp;Import-Package=net.i2p.crypto.eddsa;resolution:=optional</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector-tcp-netty/${californium-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/scandium/${californium-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-coap/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-coap' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>camel-netty</feature>
+        <feature>bouncycastle</feature>
+        <bundle>wrap:mvn:org.eclipse.californium/californium-core/${californium-version}</bundle>
```

Comment:
Why do you use the wrap protocol here? `californium-core` is an OSGI bundle so it should not be needed

Reply:
Please also add the attribute `dependency='true'` to only install the dependency if it has not yet been installed